src/Service.Tests/SqlTests/RestApiTests/Delete/DwSqlDeleteApiTest.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.SqlTests.RestApiTests.Delete
+{
+    /// <summary>
+    /// Test REST Apis validating expected results are obtained.
+    /// </summary>
+    [TestClass, TestCategory(TestCategory.DWSQL)]
+    public class DwSqlDeleteApiTests : DeleteApiTestBase
+    {
+        private static Dictionary<string, string> _queryMap = new()
+        {
+            {
+                "DeleteOneWithStoredProcedureTest",
+                $"SELECT [id] FROM { _integrationTableName } " +
+                $"WHERE id = 14"
+            }
+        };
+        #region Test Fixture Setup
+
+        /// <summary>
+        /// Sets up test fixture for class, only to be run once per test run, as defined by
+        /// MSTest decorator.
+        /// </summary>
+        [ClassInitialize]
+        public static async Task SetupAsync(TestContext context)
+        {
+            DatabaseEngine = TestCategory.DWSQL;
+            await InitializeTestFixture();
+        }
+
+        /// <summary>
+        /// Runs after every test to reset the database state
+        /// </summary>
+        [TestCleanup]
+        public async Task TestCleanup()
+        {
+            await ResetDbStateAsync();
+        }
+
+        #endregion
+
+        [TestMethod]
+        public async Task DeleteOneInViewBadRequestTest()
+        {
+            string expectedErrorMessage = $"View or function '{_defaultSchemaName}.{_composite_subset_bookPub}' is not updatable " +
+                                           "because the modification affects multiple base tables.";
+            await base.DeleteOneInViewBadRequestTest(expectedErrorMessage);
+        }
+
```

Comment:
It seems like DW tests are still missing for Patch/Put - those are required for complete REST APIs support for DW.

Reply:
we don't have these types of tests for SQL either, I propose an issue to track and follow-up because as-is, this PR is in parity with SQL. 
@rohkhann , Please check at least what the behavior is when runtime config defines stored proc to be PUT/PATCH
```json
"rest": {
  "methods": [ "GET", "POST" ]
}
```
per https://learn.microsoft.com/en-us/azure/data-api-builder/views-and-stored-procedures#rest-support-for-stored-procedures